package-lock.json

```
@@ -41108,7 +41108,7 @@
       }
     },
     "decentraland-ecs": {
-      "version": "file:ecs/decentraland-ecs-6.6.1-20201020183014.commit-bdc29ef-hotfix.tgz",
+      "version": "file:ecs\\decentraland-ecs-6.6.1-20201020183014.commit-bdc29ef-hotfix.tgz",
```

One more tiny comment. Would you mind checking out the `package-lock.json` for `master` and install only the latest version of `decentraland-ui`? Or leave this line as it was before?

```suggestion
      "version": "file:ecs/decentraland-ecs-6.6.1-20201020183014.commit-bdc29ef-hotfix.tgz",
```

---

I still see this. Would you mind pulling the last changes from `master` and leaving the `package-lock` as in the newest version?  In this scenario, we want to introduce as few changes in this file as possible. 🙏

---

@kevinszuchet  Hope this time it solves the issue!